src/main/kotlin/com/github/pool_party/pull_party_bot/commands/PingCommandHandlers.kt

```diff
@@ -173,15 +173,25 @@ private fun Bot.handlePartyChangeRequest(isNew: Boolean, msg: Message, args: Str
         return
     }
 
-    val partyName = parsedList[0].removePrefix("@")
+    val partyName = parsedList[0].replace("@", "")
+
+    if (partyName.isBlank()) {
+        sendMessage(chatId, ON_PARTY_NAME_FAIL)
+        return
+    }
 
-    // TODO name validation
     if (!modifyCommandAssertion(chatId, partyName)) {
         return
     }
 
     val users = parsedList.drop(1)
-        .map { if (!it.startsWith("@")) "@$it" else it }.distinct()
+        .map { it.replace("@", "") }.distinct()
+        .filter { it.matches("([a-z0-9_]{5,32})".toRegex()) }
+        .map { "@$it" }
+
+    if (users.size < parsedList.distinct().size - 1) {
```

Comment:
Maybe you should bring upper bound user amount validation, as well? 🤔

Reply:
> Maybe you should bring upper bound user amount validation, as well? 🤔

 	This condition checks whether the amount of valid users equals the original amount of given users, I don't understand how there can be more than a single bound

Reply:
My point was that you can add here one more validation. This assertion would check that amount of valid (given) users doesn't exceed a certain constant (so that it would be impossible to fuck our db with a request on 1m users (flexers) in one party)

Reply:
> My point was that you can add here one more validation. This assertion would check that amount of valid (given) users doesn't exceed a certain constant (so that it would be impossible to fuck our db with a request on 1m users (flexers) in one party)

I thought that 1Gb as a `Text`  field in db is enough for any request yet, especially considering telegram messages cropping

Reply:
Reasonable enough

Reply:
Probably, it would be a great idea to set a limit for amount of parties in a chat

Reply:
> Probably, it would be a great idea to set a limit for amount of parties in a chat 🤔

We will come back to this question with the amount of active chats and requests growing anyway

Reply:
Reasonable enough